README.md

```diff
@@ -1,9 +1,98 @@
 # edgex-checkbox-provider
 
-This project contains the Checkbox tests of the [EdgeX Foundry](https://docs.edgexfoundry.org/) snaps.
+## Introduction
 
-The upstream repository is hosted on Github:  
-https://github.com/canonical/edgex-checkbox-provider
+This project contains the checkbox tests of the [edgex foundry](https://docs.edgexfoundry.org/) snaps. 
+[Checkbox](https://checkbox.readthedocs.io/en/latest/) is a test automation software performed by the Canonical certification team. 
+The [upstream repository](/../../) is hosted on Github; 
```

Comment:
`/../../` this should be an absolute URL to direct people looking at Launchpad mirror to Github.